src/test/java/org/fundation/coding/adrian/katas/GinevereCyphersTest.java

```diff
@@ -0,0 +1,28 @@
+package org.fundation.coding.adrian.katas;
+
+import org.fundation.coding.adrian.katas.cyphers.GinevereCrypto;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Ginevere tests.
+ */
+public class GinevereCyphersTest {
+    /**
+     * Example tests on the specification of the kata.
+     */
+    @Test
+    public void test3() {
+        GinevereCrypto ginevereCrypto = new GinevereCrypto();

```

Comment:
You are repeating this line of code in all your tests
Please move it to a @before method.